.circleci/config.yml

```diff
@@ -244,8 +244,8 @@ workflows:
       - generate-config:
           definitions: << pipeline.parameters.config-definitions >>
           config-definitions-branches: << pipeline.parameters.config-definitions-branches >>
-            #"tests/test-definitions.txt tests/go-test-definitions.txt tests/arangojs-test-definitions.txt"
-            # tests/java-test-definitions.txt"
-            # TODO: make go driver tests work and add " tests/go-test-definitions.txt"
+            #"tests/test-definitions.yml tests/go-test-definitions.yml tests/arangojs-test-definitions.yml"
+            # tests/java-test-definitions.yml"
+            # TODO: make go driver tests work and add " tests/go-test-definitions.yml
```

Comment:
Looks like the go driver tests are already included. Is this TODO still relevant?